roles/storage-glusterfs/files/glusterfile-provisioner.yml

```diff
@@ -0,0 +1,84 @@
+---
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: glusterfile-provisioner-runner
+  labels:
+    glusterfs: file-provisioner-runner-clusterrole
+    glusterfile: provisioner-runner-clusterrole
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["services"]
+    verbs: ["get","create","delete"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["routes"]
+    verbs: ["get", "list"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get", "create","delete"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: glusterfile-provisioner
+  labels:
+    glusterfs: file-provisioner-sa
+    glusterfile: provisioner-sa
+---
+apiVersion: v1
+kind: ClusterRoleBinding
+metadata:
+  name: glusterfile-provisioner
+roleRef:
+  name: glusterfile-provisioner-runner
+subjects:
+- kind: ServiceAccount
+  name: glusterfile-provisioner
+  namespace: ${NAMESPACE}
+---
+kind: DeploymentConfig
+apiVersion: v1
+metadata:
+  name: glusterfile-provisioner-dc
+  labels:
+    glusterfs: file-provisioner-dc
+    glusterfile: provisioner-dc
+  annotations:
+    description: Defines how to deploy the glusterfile provisioner pod.
+spec:
+  replicas: 1
+  selector:
+    glusterfs: file-provisioner-pod
+  triggers:
+  - type: ConfigChange
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      name: glusterfile-provisioner
+      labels:
+        glusterfs: file-provisioner-pod
+    spec:
+      serviceAccountName: glusterfile-provisioner
+      containers:
+      - name: glusterfile-provisioner
+        image: gluster/glusterfile-provisioner:latest
```

Comment:
This should be `gluster/glusterfileclone:latest` as the clone functionality has not landed in the master branch yet. Once heketi has merged the clone support, the provisioner can merge those changes too.

Reply:
This also belongs in the dependent PR. :) Already captured there, at least.